Formula/p/pixi-pack.rb

```diff
@@ -0,0 +1,28 @@
+class PixiPack < Formula
+  desc "Pack and unpack conda environments created with pixi"
+  homepage "https://github.com/quantco/pixi-pack"
+  url "https://github.com/quantco/pixi-pack/archive/refs/tags/v0.3.1.tar.gz"
+  sha256 "d0c6eb4d20747a5ea47093dc06e80e15a46ec3fd9c3c1e93ee035480e3b0a75a"
+  license "BSD-3-Clause"
+  head "https://github.com/quantco/pixi-pack.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+
+  uses_from_macos "bzip2"
+
+  on_linux do
+    depends_on "openssl@3"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+
+    generate_completions_from_executable(bin/"pixi", "completion", "-s")
```

Comment:
the completion command does not exist?

```
$ pixi-pack completion --help
error: unrecognized subcommand 'completion'
```

Reply:
oops, copied wrong stuff https://github.com/Quantco/pixi-pack/issues/82